index.html

```diff
@@ -4,81 +4,84 @@
     </section>
-    <section class="part management">
-      <div id="management" class="part-imgbox">
-        <img alt="폴더 이름 변경하기 이미지" src="https://s3-alpha-sig.figma.com/img/169f/c357/d8b9be5bafee36b14292379248107bfd?Expires=1694995200&Signature=j-GBqcfzULv3bQvWsu9f3m9H35RmwfxYHTpSPPpiZGhhkWEgD3KneX7xu9N0X2WgJ6YGapJ6LCe9rhvE8p1dCWo14qAIeurB~AFu2DsE4DZbiuyrIcU3xSwq~I~UzXZDMwJAnS3j~rB7hvjzyYxm9SZGvMPX1eEomUoXO81Z~XAQM4AzZT9T9mVKJiGJq6TuNfs2YRU36e0dn1GJB1tuS9ZuXYMvpXUKz2Wc7hlnJSIjgMWPMyq9nNmvsNQIYVcRLJ9hHTjJwUdo2plz4vPwSMAU~uBbZyYcJrKQlmdMnz6BuatRK4nRePc03ZUrAtvd39itOI~gKvdtyg9NbC22OA__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4">
-      </div>
+    <section class="section reverse">
+      <img class="section-img" alt="폴더 이름 변경하기 이미지" src="./public/images/image2.png">
       <div class="content">
-        <h2>링크를 폴더로<br><span>관리</span>하세요</h2>
-        <p>나만의 폴더를 무제한으로 만들고<br>다양하게 활용할 수 있습니다.</p>
+        <h2 class="title">링크를 폴더로 <br><span class="management gradient">관리</span>하세요</h2>
+        <p class="description">나만의 폴더를 무제한으로 만들고 <br>다양하게 활용할 수 있습니다.</p>
       </div>
     </section>
-    <section class="part sharing">
+    <section class="section">
       <div class="content">
-        <h2>저장한 링크를<br><span>공유</span>해보세요</h2>
-        <p>여러 링크를 폴더에 담고 공유할 수<br>있습니다. 가족, 친구, 동료들에게 쉽고<br>빠르게 링크를 공유해 보세요.</p>
-      </div>
-      <div id="sharing" class="part-imgbox">
-        <p>
-          <img alt="폴더 공유하기 이미지" src="https://s3-alpha-sig.figma.com/img/acc1/c137/e4c7c24a1de11d2c66e956a2f5e51b60?Expires=1694995200&Signature=VXGv-RmC5M5h4OgVipw5k-iaOs4lAaLsiS7K4Zh1EQX5ThkE~FFKG91LgLv8jxvcpSnJws92WlgyeRnLDakLmndr0wsqCJnj4A7Kd71eNtLDfn-DP4jbMmfLIUc5ZBsQk5m3IyjZ5rtQ8XtOEYxmAydvMMH886RBipPZs6QqMHCP5zoa5fMwIZThXRCFbHLMAmhB4To2mV3nBJS6XCg8Gbe2bvzdR~UzVsZxf8vtDEP-5ViRb0rMuWkGDAgg2VXGJ3GpLjCc4aplbN0PQ4RHAeqvop4oh9KCZB1QD6GKqmxBXJWFEPFbfC4GCynbqbAdZCjcLp6IZA0I2~JIokMNEQ__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4">
-        </p>
+        <h2 class="title">저장한 링크를 <br><span class="sharing gradient">공유</span>해보세요</h2>
+        <p class="description">여러 링크를 폴더에 담고 공유할 수 <br>있습니다. 가족, 친구, 동료들에게 쉽고 <br>빠르게 링크를 공유해 보세요.</p>
       </div>
+      <img class="section-img" alt="폴더 공유하기 이미지" src="./public/images/image3.png">
     </section>
-    <section  class="part search">
-      <div id="search" class="part-imgbox">
-        <img alt="코드잇으로 검색한 결과 화면 이미지" src="https://s3-alpha-sig.figma.com/img/7a0c/dfbc/968f67f18c20cfb14b1e2e3ad8a3f125?Expires=1694995200&Signature=c6c1OIwshyFslCvYbA86PZsqG9LNBQjwgJ05d5lm6gND4r84PFiC0AmcNDZ4VAcnpLT70goJPEA4rDDIn8KBhwJssY5iC8T9KVuOxdgubqu95yA41t~G1kp9EyOE8uFAGxL2b0na947GlmXThlKziL4Hi~ECb00RXIyaw9kt7LST8ohsFRkcX0zK4KYb-JK04IfbLpkAvAfzlj76RBgHYba8KMByucAW80m4zvAyUT3WAopfxhEeM~PxY5YKKhU5nNqqyrwB5K6ZCSk32u21fgnI-~vNfHyrSVPsT1QVmbGmEt6hPUg45rCoHUqQePYQ7X3RO6H2LG0RPu0BPpSBnQ__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4" width="619px" height="584px">
-      </div>
+    <section  class="section reverse">
```

Comment:
```suggestion
    <section class="section reverse">
```

아래와 같이 포맷팅이 되면 좋을 것 같아요! 포맷팅 도구 사용하시는게 없으시다면 formatting extension 을 설치해서 도움을 받아보시면 훨씬 편하고 가독성이 좋아질거예요! 참고 부탁드릴게요!

Reply:
VScode쓰고 있는데 Prettier - Code formatter 이거 다운받으면 되는건가요?